toolchains/osx.toolchain.cmake

```diff
@@ -39,17 +39,40 @@ set( TNUN_os_suffix OSX )
 
 set( TNUN_cpu_archs default )
 
+set( TNUN_ABI x64 )
+
 add_compile_options( -mmmx -mfpmath=sse -mcx16 )
 
-#...mrmlj...todo...
-#if ( <arch> STREQUAL sse3 )
-#    set( XCODE_ATTRIBUTE_CFLAGS_i386   "-msse3  -march=prescott -mtune=core2"  )
-#    set( XCODE_ATTRIBUTE_CFLAGS_x86_64 "-mssse3 -march=core2    -mtune=corei7" )
-#elseif ( <arch> STREQUAL sse4.1 )
-#    set( XCODE_ATTRIBUTE_CFLAGS_i386   "-msse4.1 -march=core2 -mtune=core2"  )
-#    set( XCODE_ATTRIBUTE_CFLAGS_x86_64 "-msse4.1 -march=core2 -mtune=corei7" )
-#endif()
+if("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang")
```

Comment:
USING_CLANG and USING_GCC are for simplicity (to make following ifs easier to read) and can be removed.

I agree about TNUN_CPP_LIB variable which should generally be platform agnostic, but OTOH forcing to use specific STL with specific compiler requires flags specific to both compiler and platform (see line 71). This is something that should probably be defined in separate "toolchain add-on" files.